base/linalg/uniformscaling.jl

```diff
@@ -167,6 +167,8 @@ end
 
 inv(J::UniformScaling) = UniformScaling(inv(J.λ))
 norm(J::UniformScaling, p::Real=2) = abs(J.λ)
+det(J::UniformScaling{T}) where {T} = isone(J.λ) ? one(T) : throw(ArgumentError("Determinant of UniformScaling is only well-defined when λ=1."))
+logdet(::UniformScaling{T}) where {T} = isone(J.λ) ? log(one(T)) : throw(ArgumentError("Determinant of UniformScaling is only well-defined when λ=1."))
```

Comment:
Missing `J` for the argument. But we actually do not need this since we have https://github.com/JuliaLang/julia/blob/e701561bb9dd2bde921f172ce8fc2a97420d886f/base/linalg/generic.jl#L1267